### PR TITLE
handle login

### PR DESCRIPTION
**Pull request by dorinagy**

### CODE REVIEW

File: app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java [hunk 1]
```diff
@@ -139,28 +124,57 @@ public void initialize(HttpRequest request) throws IOException {
...
+      @Override
+      public void onFailure(Call<LoginResponse> call, Throwable t) {
+        Toast.makeText(LoginActivity.this, "login error", Toast.LENGTH_SHORT).show();
+        Log.d("login", "FAIL! =(");
+      }
+    });
+  }
+
+  protected void saveLoginData(String username, String token) {
+    editor.putString("Token", token);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java]: @dorinagy use constants for the key

File: app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java [hunk 2]
```diff
@@ -139,28 +124,57 @@ public void initialize(HttpRequest request) throws IOException {
...
+    editor.putString("Username", username);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java]: @dorinagy here as well

File: app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java [hunk 3]
```diff
@@ -139,28 +124,57 @@ public void initialize(HttpRequest request) throws IOException {
...
     return controller;
   }
+
+  public void onLogin(final String username, final String token) {
+    LoginRequest testLogin = new LoginRequest(username, token);
+    service.login(testLogin).enqueue(new Callback<LoginResponse>() {
+      @Override
+      public void onResponse(Call<LoginResponse> call, Response<LoginResponse> response) {
+        if (response.body().getStatus().equals("ok")) {
+          Log.d("dev", response.body().getToken());
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java]: @dorinagy remove the logging of the actual token

File: app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java [hunk 4]
```diff
@@ -33,68 +34,52 @@
...
+    setContentView(R.layout.activity_login);
+
+    GitinderApp.app().basicComponent().inject(this);
+
+    ActionBar actionBar = getSupportActionBar();
+    actionBar.setDisplayShowHomeEnabled(true);
+
+    editor = preferences.edit();
+
+    onLogin("Bond", "abcd1234");
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java]: @dorinagy nono, where do we know the github access token?

File: app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java [hunk 5]
```diff
@@ -33,68 +34,52 @@
...
+    onLogin("", "");
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/LoginActivity.java]: why?

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by Garlyle — APPROVED** [review]

**Pull request merged**